Kwave has private libraries for its plugins, which don't really need the .so
symlink. Thus, just tell CMake to not install them.

--- a/libgui/CMakeLists.txt	2008-09-20 08:07:23.000000000 +0200
+++ b/libgui/CMakeLists.txt	2009-04-02 22:25:56.000000000 +0200
@@ -76,7 +76,7 @@
 #############################################################################
 ### install files                                                         ###
 
-INSTALL(TARGETS libkwavegui LIBRARY DESTINATION ${KDE4_LIB_INSTALL_DIR})
+INSTALL(TARGETS libkwavegui LIBRARY DESTINATION ${KDE4_LIB_INSTALL_DIR} NAMELINK_SKIP)
 
 #############################################################################
 #############################################################################
--- a/libkwave/CMakeLists.txt	2008-12-23 13:32:06.000000000 +0100
+++ b/libkwave/CMakeLists.txt	2009-04-02 22:25:30.000000000 +0200
@@ -119,7 +119,8 @@
 #############################################################################
 ### install files                                                         ###
 
-INSTALL(TARGETS libkwave LIBRARY DESTINATION ${KDE4_LIB_INSTALL_DIR})
+INSTALL(TARGETS libkwave LIBRARY DESTINATION ${KDE4_LIB_INSTALL_DIR} NAMELINK_SKIP)
+
 
 #############################################################################
 #############################################################################
